Include/pyport.h

```diff
@@ -663,6 +663,27 @@ extern char * _getpty(int *, int, mode_t, int);
 #  define WITH_THREAD
 #endif
 
+#ifdef WITH_THREAD
```

Comment:
This test is useless. This macro is now always defined. It's only kept for backward compatibility.

Reply:
Doesn't it affect WASM builds?

Reply:
See the code 3 lines above:

```c
#ifndef WITH_THREAD
#  define WITH_THREAD
#endif
```